README.md

```diff
@@ -227,8 +227,8 @@ Default: `process.cwd()`
 Ability to set the current working directory. Defaults to `process.cwd()`. Can either be a string to set the cwd to match files and spawn tasks. Or an object to set each independently. Such as `options: { cwd: { files: 'match/files/from/here', spawn: 'but/spawn/files/from/here' } }`.
 
 #### options.livereloadOnError
-Type: `Boolean`  
-Default: `true`  
+Type: `Boolean`
```

Comment:
I didn't make those changes, probably got stripped automatically. In either case, this can be closed, just update to the latest tiny-lr and see https://github.com/mklabs/tiny-lr/issues/96